CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## 32 Release Notes
 
+### Important Notes
+
+Cromwell instances configured to interact with **Google Cloud Storage** in any fashion, including but not limited to configurations with a **Pipelines API (PAPI) backend** (formerly JES backend) should **read the following section carefully**:
```

Comment:
Is it possible that in Changelog we basically limit the main points to  
*If* you access files in GCS *then* you now have to do ...

And then abstract away most of the rest to the new sections you've written up in the `Google.md` file? The explanation of "why" this is required is more of an implementation detail. I do think certain users would like to understand the reason behind this better --  but most people will need to know *what* and *how* to make the required changes.

Reply:
Ok